src/versionbits.cpp

```diff
@@ -159,26 +36,20 @@ ThresholdState AbstractThresholdConditionChecker::GetStateFor(const CBlockIndex
     int64_t nActualLockinTime = 0;
     int nActualLockinBlock = 0;
 
-    if (nPeriod == 0)
-    {
-        // we cannot do anything further -this deployment is not really defined.
-        return THRESHOLD_DEFINED;
-    }
+    assert(nPeriod != 0);
```

Comment:
Will this condition always be caught before a deployment is published?  Elsewhere you removed the requirement that windowsize set for a deployment to be considered configured.

Reply:
OK, we may need to keep either one (not assert OR  not consider windowsize == 0 to be configured)